### PR TITLE
Updating docs

### PR DESCRIPTION
**Pull request by skwalker**
Documentation for CountVariants

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java [hunk 1]
```diff
@@ -9,15 +9,33 @@
...
+ *      -V input_variants.vcf
+ * </pre>
+ */
 @DocumentedFeature
 @CommandLineProgramProperties(
-        summary = "Walks over the input data set, calculating the number of variants seen.",
-        oneLineSummary = "Count variants in a VCF file",
+        summary = CountVariants.USAGE_SUMMARY,
+        oneLineSummary = CountVariants.USAGE_ONE_LINE_SUMMARY,
         programGroup = VariantProgramGroup.class
```
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java]: Program group should be VariantEvaluationProgramGroup.class

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java [hunk 2]
```diff
@@ -9,15 +9,33 @@
 import org.broadinstitute.hellbender.engine.ReferenceContext;
 import org.broadinstitute.hellbender.engine.VariantWalker;
 
+/**
+ *
+ * Count variants in a VCF file.
```
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java]: --> Counts variant records in a VCF file, regardless of filter status. The tool gives the count at end of the standard out.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java [hunk 3]
```diff
@@ -9,15 +9,33 @@
...
+ *
+ * <h3> Input </h3>
+ * <ul>
+ *     <li> A single VCF file. </li>
+ * </ul>
+ *
+ * <h3> Example: </h3>
```
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java]: Example: --> Usage example

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java [hunk 4]
```diff
@@ -9,15 +9,33 @@
...
-        summary = "Walks over the input data set, calculating the number of variants seen.",
-        oneLineSummary = "Count variants in a VCF file",
+        summary = CountVariants.USAGE_SUMMARY,
+        oneLineSummary = CountVariants.USAGE_ONE_LINE_SUMMARY,
         programGroup = VariantProgramGroup.class
 )
 public final class CountVariants extends VariantWalker{
     private long count = 0;
 
+    static final String USAGE_ONE_LINE_SUMMARY = "Count variants in a VCF file";
```
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java]: Count variants --> Counts variant records

Remember that variants can be at the allele level and what this tool counts is the number of rows and which does not necessarily reflect the number of variant alleles.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java [hunk 5]
```diff
@@ -9,15 +9,33 @@
...
+    static final String USAGE_SUMMARY = "Walks over the input data set, calculating the number of variants seen.";
```
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java]: Walks over the input data set, calculating the number of variants seen. --> Counts variant records in a VCF file, regardless of filter status.

**Comment by sooheelee** [comment]
Back to you @skwalker.

**Comment by skwalker** [comment]
@sooheelee thanks for the review! Want to take one final glance before I merge?

**Comment by sooheelee** [comment]
Thanks for incorporating the suggestions. Looks great @skwalker.

**Pull request merged**